src/sass_types/sass_value_wrapper.h

```diff
@@ -88,6 +92,65 @@ namespace SassTypes
   }
 
   template <class T>
+  NAN_METHOD(SassValueWrapper<T>::Operation) {
+    if (info.Length() < 2) {
+      return Nan::ThrowTypeError("Not enough arguments for operation");
+    }
+    if (!info[0]->IsString()) {
+      return Nan::ThrowTypeError("Operation value should be a string");
+    }
+    if (SassTypes::Value *rhs = SassTypes::Factory::unwrap(info[1])) {
```

Comment:
`unwrap` generates whatever it can, currently only non-native artificial SassTypes. https://github.com/sass/node-sass/pull/1310 is there to introduce proper primitive JavaScript null, I have also a patch to introduce native `true` and `false`. 

Numbers are more difficult, since Sass number is actually a `(number, unit)` tuple and JavaScript does not support that.